src/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ elseif(MSVC)
     target_compile_options(VulkanCompilerConfiguration INTERFACE
         /W4
         /we5038 # Enable warning about MIL ordering in constructors
+	/wd4324 # Disable warning about alignment padding
```

Comment:
Normally I'd be hesitant to add warnings as this target is intended to be consumed downstream. HOWEVER 1. I don't think anyone uses it and 2. this w4324 seems to be a real nonsense warning due to alignment.

Reply:
this warning comes from vku::concurrent::unordered_map which purposefully adds padding for better multithreaded performance.